[refresh2020q1][libgpg-error] - Make error while building libgpg-error

This error is occuring during the refresh process. I have posted the context of the error and linked the file at the bottom of the issue.

Error:

```
gawk: fatal: cannot use gawk builtin `namespace' as variable name
make[2]: *** [Makefile:1617: errnos-sym.h] Error 2
make[2]: Leaving directory '/hab/cache/src/libgpg-error-1.36/src'
make[1]: *** [Makefile:509: all-recursive] Error 1
make[1]: Leaving directory '/hab/cache/src/libgpg-error-1.36'
make: *** [Makefile:441: all] Error 2
   libgpg-error: Build time: 0m9s
   libgpg-error: Exiting on error
```

Steps to reproduce:

```
cd /tmp/
git clone --depth 1 -b refresh/2020q1_008 --single-branch https://github.com/habitat-sh/core-plans.git /tmp/core-plans"
wget https://gist.githubusercontent.com/MindNumbing/255ca89882213bf0a4422f28fb902570/raw/b00ed3638b73d8a697ca411ea5154bd77118d456/hab_studio.env
source hab.env
hab studio enter
build core-plans/libgpg-error
```

Full Error Log:
[libgpg-error.txt](https://github.com/habitat-sh/core-plans/files/4458514/libgpg-error.txt)

--- a/.refresh/remaining_plans_failed.txt
+++ b/.refresh/remaining_plans_failed.txt
@@ -1,6 +1,5 @@
 core-plans/libnsl
 core-plans/grpcurl
-core-plans/libgpg-error
 core-plans/libassuan
 core-plans/gpgme
 core-plans/gox

--- a/.refresh/remaining_plans_passed.txt
+++ b/.refresh/remaining_plans_passed.txt
@@ -158,6 +158,7 @@ core-plans/libpthread-stubs
 core-plans/gsl
 core-plans/grafana-loki
 core-plans/grafana
+core-plans/libgpg-error
 core-plans/graphviz
 core-plans/gperftools
 core-plans/govc

--- a/libgpg-error/plan.sh
+++ b/libgpg-error/plan.sh
@@ -1,12 +1,12 @@
 pkg_name=libgpg-error
 pkg_origin=core
-pkg_version=1.36
+pkg_version=1.37
 pkg_license=('GPL-2.0-or-later')
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_source="ftp://ftp.gnupg.org/gcrypt/${pkg_name}/${pkg_name}-${pkg_version}.tar.bz2"
 pkg_upstream_url="https://www.gnupg.org/software/libgpg-error/index.html"
 pkg_description="Libgpg-error is a small library that originally defined common error values for all GnuPG components."
-pkg_shasum=babd98437208c163175c29453f8681094bcaf92968a15cafb1a276076b33c97c
+pkg_shasum=b32d6ff72a73cf79797f7f2d039e95e9c6f92f0c1450215410840ab62aea9763
 pkg_deps=(core/glibc)
 pkg_build_deps=(
   core/gcc

